docs/api.md

```diff
@@ -4024,7 +4164,9 @@ These are in alphabetic order, with the exception that the `Extended` versions o
 ## Form Attachment (object)
 + name: `myfile.mp3` (string, required) - The name of the file as specified in the XForm.
 + type: (Form Attachment Type, required) - The expected type of file as specified in the XForm.
-+ exists: `true` (boolean, required) - Whether the server has the file or not.
++ exists: `true` (boolean, required) - True if the Attachment is linked to a Dataset or the server has the file otherwise false.
```

Comment:
"True if the server has the file or the Attachment is linked to a Dataset, otherwise false."